apps/pie-docs/src/_11ty/shortcodes/categorisedIconList.js

```diff
@@ -12,9 +12,18 @@ const pieIcons = require('../filters/pieIconsSvg')();
 const buildIconCard = (icon) => {
     const pieIcon = pieIcons.find((x) => x.name === icon.name).icon;
```

Comment:
I think this line will need optional chaining too
```suggestion
    const pieIcon = pieIcons.find((x) => x.name === icon.name)?.icon;
```